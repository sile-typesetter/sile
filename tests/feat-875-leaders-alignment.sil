\begin[papersize=a5]{document}
\script[src=packages/leaders]
\nofolios
\neverindent
% Test case: vertical alignement for \leaders with custom pattern
\font[size=50pt]
|\leaders{/}|\par
\hbox{}\leaders{/}\hbox{}\par
1\leaders{/}1\par
1 2\leaders{/}2 1\par
1 2 3\leaders{/}3 2 1\par
1 2 3 4\leaders{/}4 3 2 1\par
1 2 3 4 5\leaders{/}5 4 3 2 1\par
\end{document}
