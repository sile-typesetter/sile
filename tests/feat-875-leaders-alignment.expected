Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.9764
My 	70.5353
Set font 	Gentium Plus;50;400;;normal;;LTR
T	95 w=12.2070	(|)
Mx 	46.6225
T	18 w=23.4619	(/)
Mx 	70.0844
T	18 w=23.4619	(/)
Mx 	93.5463
T	18 w=23.4619	(/)
Mx 	117.0082
T	18 w=23.4619	(/)
Mx 	140.4701
T	18 w=23.4619	(/)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	257.7797
T	18 w=23.4619	(/)
Mx 	281.2416
T	18 w=23.4619	(/)
Mx 	304.7035
T	18 w=23.4619	(/)
Mx 	328.1654
T	18 w=23.4619	(/)
Mx 	351.6274
T	18 w=23.4619	(/)
Mx 	386.3442
T	95 w=12.2070	(|)
Mx 	20.9764
My 	130.5353
Mx 	23.1606
T	18 w=23.4619	(/)
Mx 	46.6225
T	18 w=23.4619	(/)
Mx 	70.0844
T	18 w=23.4619	(/)
Mx 	93.5463
T	18 w=23.4619	(/)
Mx 	117.0082
T	18 w=23.4619	(/)
Mx 	140.4701
T	18 w=23.4619	(/)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	257.7797
T	18 w=23.4619	(/)
Mx 	281.2416
T	18 w=23.4619	(/)
Mx 	304.7035
T	18 w=23.4619	(/)
Mx 	328.1654
T	18 w=23.4619	(/)
Mx 	351.6274
T	18 w=23.4619	(/)
Mx 	375.0893
T	18 w=23.4619	(/)
Mx 	398.5512
Mx 	20.9764
My 	190.5353
T	20 w=23.4619	(1)
Mx 	46.6225
T	18 w=23.4619	(/)
Mx 	70.0844
T	18 w=23.4619	(/)
Mx 	93.5463
T	18 w=23.4619	(/)
Mx 	117.0082
T	18 w=23.4619	(/)
Mx 	140.4701
T	18 w=23.4619	(/)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	257.7797
T	18 w=23.4619	(/)
Mx 	281.2416
T	18 w=23.4619	(/)
Mx 	304.7035
T	18 w=23.4619	(/)
Mx 	328.1654
T	18 w=23.4619	(/)
Mx 	351.6274
T	18 w=23.4619	(/)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	250.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	93.5463
T	18 w=23.4619	(/)
Mx 	117.0082
T	18 w=23.4619	(/)
Mx 	140.4701
T	18 w=23.4619	(/)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	257.7797
T	18 w=23.4619	(/)
Mx 	281.2416
T	18 w=23.4619	(/)
Mx 	304.7035
T	18 w=23.4619	(/)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	310.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	140.4701
T	18 w=23.4619	(/)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	257.7797
T	18 w=23.4619	(/)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	370.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	131.0008
T	23 w=23.4619	(4)
Mx 	163.9320
T	18 w=23.4619	(/)
Mx 	187.3940
T	18 w=23.4619	(/)
Mx 	210.8559
T	18 w=23.4619	(/)
Mx 	234.3178
T	18 w=23.4619	(/)
Mx 	265.0649
T	23 w=23.4619	(4)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	430.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	131.0008
T	23 w=23.4619	(4)
Mx 	167.6756
T	24 w=23.4619	(5)
Mx 	228.3901
T	24 w=23.4619	(5)
Mx 	265.0649
T	23 w=23.4619	(4)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
End page
Finish
